src/dotnet/Program.cs

```diff
@@ -178,6 +182,9 @@ private static void ConfigureDotNetForFirstTimeUse()
                             environmentProvider);
 
                         dotnetConfigurer.Configure();
+
+                        if (!File.Exists(Telemetry.TelemetrySentinel))
+                            File.Create(Telemetry.TelemetrySentinel);
```

Comment:
Hum, I see. It makes sense too. However, because the EULA is clubbed with nuget cache priming, if the user sets the SKIP_FIRST_RUN flag, he won't get the EULA and we will still create the telemetry sentinel, right? Not sure that this is the correct behavior.

Maybe we should keep the EULA and its sentinel together. Be it with the nuget prime sentinel, or we separate both. What do you think?